openmc/deplete/results_list.py

```diff
@@ -192,3 +194,78 @@ def get_depletion_time(self):
         for ix, res in enumerate(items):
             times[ix] = res.proc_time
         return times
+
+    def export_to_materials(self, burnup_index, nuc_with_data=None):
+        """Converts a depletion time point composition into an openmc.Materials
+        instance, useful for running transport calculations on individual
+        depletion time points
+
+        This requires a materials.xml file to be present in the current
+        directory. This should correspond to the materials initially
+        present in the depletion calculation.
+
+        Only materials found in the original materials.xml file will be
+        updated. If for some reason you have modified OpenMC to produce
+        new materials as depletion takes place, this method will not
+        work as expected and leave out that material.
+
+        Parameters
+        ----------
+        burn_index : int
+            Index of burnup step to evaluate
+        nuc_with_data : Iterable of str, optional
+            nuclides iterable to evaluate with neutron data
+
+            This must be specified because not all nuclides appearing in
+            depletion results have associated neutron cross sections, and
+            as such cannot be used in subsequent transport calculations.
+            If not provided, nuclides from the cross_sections element of
+            materials.xml will be used. If that element is not present,
+            nuclides from OPENMC_CROSS_SECTIONS will be used.
```

Comment:
> iterable of str, optional

and

> This must be specified

seem to disagree

Reply:
Also, can you format this block as a single paragraph?

Reply:
The first thing only required changing the first instance of the word "not" to "if"

also did the single paragraph formatting